day_two_guide/topics/etcd_backup.adoc

```diff
@@ -0,0 +1,1167 @@
+////
+etcd tasks
+
+Module included in the following assemblies:
+
+* day_two_guide/host_level_tasks.adoc
+////
+
+etcd is the key value store for all object definitions, as well as the
+persistent master state. Other components watch for changes, then bring
+themselves into the desired state.
+
+{product-title} versions prior to 3.5 used etcd version 2, while 3.5 and later
+use v3. The data model between the two versions of etcd is different. etcd v3
+can use both the v2 and v3 data model, whereas etcd v2 can only use the v2 data
```

Comment:
If you're going to use v2 and v3 then make sure to put that in parens above next to the first spelled out occurrences.